src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -54,10 +40,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException {
     try {
       String url = request.getParameter(URL_PARAMETER);
-      CommentThreadListResponse commentResponse = generateYouTubeRequest(url).execute();
+      // TODO: add numComments parameter. Comment count will be hard-coded until then.
+      List<CommentThread> commentThreads = new YouTubeCommentRetriever().retrieveComments(url, 100);
```

Comment:
NIT:

Generally any constants should be be defined as `private static final` constants at the top of the class.

Reply:
Which variable are you referring to? URL_PARAMETER already is private static final.

Reply:
I think he is referring to the value 100.

Reply:
Oh okay, that makes sense, I only had it hard-coded like that since it was intended to be temporary until a proper variable could be passed.